### PR TITLE
Add Builds page

### PR DESCRIPTION
**Pull request by chrisrohr**

### CODE REVIEW

File: service/src/main/java/org/kiwiproject/champagne/dao/BuildDao.java [hunk 1]
```diff
@@ -4,72 +4,48 @@
...
-            // Version only filter
-            return findPagedBuildsWithVersionFilter(offset, limit, systemId, f(LIKE_QUERY_FORMAT, componentVersionFilter));
-        } else if (isBlank(componentVersionFilter)) {
-            // Name only filter
-            return findPagedBuildsWithIdentiferFilter(offset, limit, systemId, f(LIKE_QUERY_FORMAT, componentIdentifierFilter));
         }
 
         // Both name and version filtering
-        return findPagedBuildsWithVersionAndIdentifierFilters(offset, limit, systemId, componentIdentifierFilter, componentVersionFilter);
+        return findPagedBuildsWithVersionOrIdentifierFilters(offset, limit, systemId, f(LIKE_QUERY_FORMAT, componentFilter));
```
Comment (sleberknight) [inline service/src/main/java/org/kiwiproject/champagne/dao/BuildDao.java:25]: So, could there be a SQL injection hiding here?

File: service/src/main/java/org/kiwiproject/champagne/resource/BuildResource.java [hunk 2]
```diff
@@ -21,6 +14,13 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
+import com.codahale.metrics.annotation.ExceptionMetered;
```
Comment (sleberknight) [inline service/src/main/java/org/kiwiproject/champagne/resource/BuildResource.java:17]: Your import order keeps changing...we want the java ones to come _last_

File: service/src/test/java/org/kiwiproject/champagne/dao/BuildDaoTest.java [hunk 3]
```diff
@@ -97,17 +99,14 @@ void shouldInsertBuildSuccessfully() {
...
-            "null, null",
-            "null, 42.0",
-            "champagne-service, null",
-            "champagne-service, 42.0"
-        })
-        void shouldReturnListOfBuilds(String componentIdentifierFilter, String componentVersionFilter) {
+        @NullSource
+        @BlankStringSource
+        @ValueSource(strings = { "42.0", "champagne-service" })
+        void shouldReturnListOfBuilds(String componentFilter) {
```
Comment (sleberknight) [inline service/src/test/java/org/kiwiproject/champagne/dao/BuildDaoTest.java:105]: Wait...can you do this??? Stack different sources like this? Did you verify this works???

Comment (sleberknight) [reply service/src/test/java/org/kiwiproject/champagne/dao/BuildDaoTest.java:105]: Yes, you can! No idea how I never knew this, but `ArgumentsSource` is _repeatable_

File: service/src/main/java/org/kiwiproject/champagne/dao/BuildDao.java (hunk 1 above)
Comment (chrisrohr) [reply service/src/main/java/org/kiwiproject/champagne/dao/BuildDao.java:25]: It shouldn't be as we are still using JDBI's argument replacer

**Review by sleberknight — APPROVED** [review]

**Pull request merged**